docs-src/stargate-develop/modules/develop/examples/docapi/library/curl-collection-get-json-schema.sh

```diff
@@ -0,0 +1,3 @@
+curl -L -X GET '{base_doc_url}{base_doc_api}/{namespace}/collections/{collection2}/json-schema' \
```

Comment:
A note should probably be added that JSON schema support is more experimental, and that partial updates aren't allowed with a JSON schema attached

Reply:
[NOTE]
====
JSON schema support is experimental.
Also, partial updates of data are not allowed if JSON schema is defined.
====